suppliers/src/UI/embc-supplier/package.json

```diff
@@ -9,46 +9,46 @@
     "build": "ng build",
     "test": "ng test",
     "lint": "ng lint",
-    "e2e": "ng e2e",
-    "postinstall": "ngcc"
+    "format:write": "prettier . --write",
+    "e2e": "ng e2e"
   },
   "private": true,
   "dependencies": {
-    "@angular/animations": "^14.2.4",
-    "@angular/cdk": "^14.2.3",
-    "@angular/common": "^14.2.4",
-    "@angular/compiler": "^14.2.4",
-    "@angular/core": "^14.2.4",
-    "@angular/forms": "^14.2.4",
-    "@angular/localize": "^14.2.4",
-    "@angular/material": "^14.2.3",
-    "@angular/platform-browser": "^14.2.4",
-    "@angular/platform-browser-dynamic": "^14.2.4",
-    "@angular/router": "^14.2.4",
-    "@bcgov/bootstrap-theme": "https://github.com/bcgov/bootstrap-theme/releases/download/v1.1.3/bcgov-bootstrap-theme-1.1.3.tgz",
+    "@angular/animations": "^17.3.9",
+    "@angular/cdk": "^17.3.9",
+    "@angular/common": "^17.3.9",
+    "@angular/compiler": "^17.3.9",
+    "@angular/core": "^17.3.9",
+    "@angular/forms": "^17.3.9",
+    "@angular/localize": "^17.3.9",
+    "@angular/material": "^17.3.9",
+    "@angular/platform-browser": "^17.3.9",
+    "@angular/platform-browser-dynamic": "^17.3.9",
+    "@angular/router": "^17.3.9",
+    "@bcgov/bootstrap-v5-theme": "^1.0.1",
     "@fortawesome/fontawesome-free": "^5.13.0",
-    "@ng-bootstrap/ng-bootstrap": "^13.0.0",
-    "angular-oauth2-oidc": "^10.0.3",
+    "@ng-bootstrap/ng-bootstrap": "^16.0.0",
+    "angular-oauth2-oidc": "^17.0.2",
     "bootstrap": "^4.3.1",
```

Comment:
Should we not upgrade the bootstrap package as well ?

Reply:
Created EMBCESSMOD-5432 to address this, for now this stayes with V4